include/linker/common-rom.ld

```diff
@@ -172,6 +172,7 @@
 	} GROUP_LINK_IN(ROMABLE_REGION)
 #endif /* CONFIG_PCIE */
 
+#if !defined(CONFIG_LOG_CUSTOM_SECTION)
```

Comment:
Have you thought about how and whether this should integrate with the armclang linker generation support? I don't see any discussion of this in the commit message or PR description.

Reply:
@mbolivar-nordic, I am not aware of how armclang linker generation works. What is the link between that and `common-rom.ld`?  I'll be glad to bring this to discussion if any.  

`CONFIG_LOG_CUSTOM_SECTION` is default n, which won't make any difference to platforms using `common-rom.ld`.

Reply:
@mbolivar-nordic closing this thread as I did not follow this approach.